Chap_API_Server.tex

```diff
@@ -1133,13 +1140,14 @@ \subsection{\code{PMIx_server_dmodex_request}}
 \argin{cbdata}{Data to be passed to the callback function (memory reference)}
 \end{arglist}
 
-Returns one of the following:
+% TODO: Does this allow for PMIX_OPERATION_SUCCEEDED
```

Comment:
Keeping the explicit `PMIX_OPERATION_SUCCEEDED` adds the `\refarg{cbfunc} will not be called` clause which I don't think it covered by `returnsimplenb`. So I would prefer to keep it.

Reply:
In this case I think PMIX_OPERATION_SUCCEEDED should not have been added as a possible return  value because the implementation MUST call the cbfunc as it is the only way to provide the data it is being asked to look up.
(Note to self: need to remove this, which probably kills the whole itemized list)

Reply:
Yeah, @dsolt is correct - there is no way this function can return "operation succeeded". It must either return "success" (with final result in the cbfunc) or an error indicating that it cannot be processed (and the cbfunc will not be called).